scipy/stats/_continuous_distns.py

```diff
@@ -591,26 +592,31 @@ def _pdf(self, x, a, b):
         #                     gamma(a+b) * x**(a-1) * (1-x)**(b-1)
         # beta.pdf(x, a, b) = ------------------------------------
         #                              gamma(a)*gamma(b)
-        return np.exp(self._logpdf(x, a, b))
+        return _boost._beta_pdf(x, a, b)
 
     def _logpdf(self, x, a, b):
```

Comment:
@mdhaber I did not remove the  existing logpdf implementation.  If needed, I can remove this and simply call `np.log(self._pdf(x, a, b))`

Reply:
Gotcha. Actually, if you remove it entirely, I think `rv_continuous` will do that for you. But I'm not sure which way to go on this yet. Let's see if @WarrenWeckesser would you prefer to stick with Boost to ensure that `_pdf` and `_logpdf` always agree, or do we need to try to test which tends to be more accurate?

Reply:
We need to maintain `_logpdf`, otherwise we lose precision in cases where `_pdf` underflows to 0 but `_logpdf` computes an accurate value. E.g.
```
In [42]: beta.pdf(0.99995, 0.1, 80)  # Value is smaller than can be represented with float64                                                                                            
Out[42]: 0.0

In [43]: beta.logpdf(0.99995, 0.1, 80)                                                                                          
Out[43]: -784.1905450736676
```
This is important for code such as MLE fitting, where `logpdf` is repeatedly computed for each data value as the search parmeters vary.

Reply:
Okay, I'll leave the `_logpdf` implementations untouched -- good to know!